### PR TITLE
Document custom sets of genes with merged gene tracks

### PR DESCRIPTION
**Pull request by oplantalech**
Improve documentation for customizing the sets of genes from the query page, including an example for the recently added merged gene tracks.

### CODE REVIEW

File: docs/portal.properties-Reference.md [hunk 1]
```diff
@@ -134,7 +134,34 @@ oncoprint.hide_vus.default=true|false
...
+}]
+```
+
+## Example with merged gene tracks
+```
+[{
+	"id": "BRCA genes test",
+	"genes": ["[\\\"BRCA genes\\\" BRCA1: MUT=E1258D; BRCA2: HOMDEL MUT=NONSENSE MUT=NONSTART MUT=NONSTOP MUT=FRAMESHIFT MUT=SPLICE MUT=TRUNC;]"]
+}]
+```
```
Comment (pieterlukasse) [inline docs/portal.properties-Reference.md]: @oplantalech maybe for each example, state what the result will be? E.g. in the first example the result will be 2 sets of genes (so 2 items in the drop down), overwriting the current list, right? Or will these 2 items be appended to the end of the list?

Comment (oplantalech) [reply docs/portal.properties-Reference.md]: @pieterlukasse I've improved the documentation

Comment (pieterlukasse) [reply docs/portal.properties-Reference.md]: please separate different genes into separate list items. See also https://github.com/cBioPortal/cbioportal/issues/4385. So 
```
[{
	"id": "BRCA genes test",
	"genes": ["[\\\"BRCA genes\\\" BRCA1: MUT=E1258D;", "BRCA2: HOMDEL MUT=NONSENSE MUT=NONSTART MUT=NONSTOP MUT=FRAMESHIFT MUT=SPLICE MUT=TRUNC;]"]
}]
```

**Comment by oplantalech** [comment]
@jjgao can you merge this PR?

**Pull request merged**